apps/studio/pages/project/[ref]/database/backups/restore-to-new-project.tsx

```diff
@@ -99,6 +99,13 @@ const RestoreToNewProject = () => {
     {
       refetchInterval,
       refetchOnWindowFocus: false,
+      retry: (failureCount, error) => {
+        // If the user doesn't have permission to read physical backups (or they are disabled), don't retry
+        // This way we don't have to wait for 3 requests to finish.
+        if (typeof error === 'object' && error !== null && 'code' in error && error?.code === 403)
+          return false
+        return failureCount < 3
```

Comment:
just wondering - are we able to check permissions before triggering this request via the `enabled` flag?
so we're able to pre-emptively decide when to fire the request